File: Python/seleccion.py
import time
inicio = time.time()
def insertarNumero():
    lista=[]
    while True:
        n=int(input("Ingresa un numero (0 terminar):"))
        if n==0:
            return lista
        else:
            lista.append(n)
def ordenPorSeleccion(lista):
    tamaño=len(lista)
    for i in range(0,tamaño):
        min=i
        for j in range(i+1,tamaño):
            if lista[min]>lista[j]:
                min=j
        aux=lista[i]
        lista[i]=lista[min]
        lista[min]=aux
    return lista
def mostrarLista(lista):
    for numero in lista:
        print(numero)

lista=insertarNumero()
ordenPorSeleccion(lista)
print("Lista ordenada por seleccion: ")
mostrarLista(lista)
fin = time.time()
print(fin-inicio) 

File: C++/Seleccion.cpp
#include <iostream>
using namespace std;

int main()
{
    int tam, aux;
    cin >> tam;
    int arreglo[tam];

    for (int i = 0; i < tam; i++)
    {
        cin >> arreglo[i];
    }

    for (int i = 0; i < tam - 1; ++i)
    {
        int min = i;
        for (int c = i + 1; c < tam; ++c)
        {
            if (arreglo[min] > arreglo[c])
                min = c;
        }
        aux = arreglo[i];
        arreglo[i] = arreglo[min];
        arreglo[min] = aux;
    }

    for (int i = 0; i < tam; i++)
    {
        cout << arreglo[i] << " ";
    }
}